osgi.specs/docbook/160/service.jakartaws.xml

```diff
@@ -355,7 +355,55 @@ public class SOAPLoggingHandler implements SOAPHandler&lt;SOAPMessageContext&gt;
             </section>
           </section>
                 
+  <section>
+    <title>Capabilities</title>
+
+    <section>
+      <title>osgi.implementation Capability</title>
+
+      <para>The Jakarta™ XML Web Services Whiteboard Implementation bundle must provide the <link
+      linkend="service.namespaces-osgi.implementation.namespace"><code>osgi.implementation</code></link>
+      capability with the name <xref
+      linkend="org.osgi.service.webservice.whiteboard.WebserviceWhiteboardConstants.WEBSERVICE_IMPLEMENTATION"
+      xrefstyle="hyperlink"/>. This capability can be used by provisioning
+      tools and during resolution to ensure that a Typed Event implementation
+      is present. The capability must also declare a uses constraint for the
+      <code>org.osgi.service.typedevent</code> package and provide the version
```

Comment:
I think that this should be the packages `org.osgi.service.webservice.whiteboard`, `jakarta.xml.ws` and `jakarta.xml.ws.handler`. We possibly also need `jakarta.jws` and `jakarta.jws.soap` but I'm not so sure of those